besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/GenerateBlockchainConfig.java

```diff
@@ -136,8 +136,8 @@ private void checkPreconditions() {
   /** Generates output directory with all required configuration files. */
   private void generateBlockchainConfig() {
     try {
-      handleOutputDirectory();
       parseConfig();
+      handleOutputDirectory();
```

Comment:
I would postpone adding this to the PR (unless we end up modifying besu operator in this PR as well). This change would be better isolated in it's own PR so we don't have to worry about testing alongside the other stuff.

Reply:
Definitely